.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,100 @@
+name: Playwright Smoketest
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6 * * *' # nightly at 06:00 UTC against main
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
```

Comment:
Do we still need this `pull_request` trigger? This would break on PRs from forks, which cannot access secrets. We also don't want to risk this secret getting exposed to forks anyway.